src/views/credentials/viewCredentialsList.tsx

```diff
@@ -241,13 +241,14 @@ const CredentialsListView: React.FunctionComponent = () => {
             <EmptyState>
               <EmptyStateHeader
                 headingLevel="h4"
-                titleText="No available credential"
+                titleText="No available credentials"
                 icon={<EmptyStateIcon icon={PlusCircleIcon} />}
               />
               <EmptyStateBody>
-                A credential contains authentication information needed to scan a source.A credential includes a
-                username and a password or SSH key. The quipucords tool uses SSH to connect to servers on the network
-                and uses credentials to access those servers.
+                Begin by adding credentials. Credentials contain the authentication information required to access and
+                scan a source. This includes a username, password, SSH key, or API token. The system can use these
+                credentials either to connect via SSH or to authenticate directly with system APIs, enabling access to
+                the necessary environments and systems.
```

Comment:
optional/future:
move strings into the locale file since we already handle strings in this component